### ISSUE TITLE
Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag

### ISSUE THREAD

**Issue by Aladdin-ADD** [vuejs/eslint-plugin-vue#188]
**Tell us about your environment**

* **ESLint Version:** v4.7.1
* **eslint-plugin-vue Version:** 3.10.0
* **Node Version:** v8.4.0

**Please show your full configuration:**
[@yonyou-cloud/eslint-config](https://github.com/yonyou-cloud/eslint-config/blob/master/vue.js)

```
module.exports = {
  extends: ['./base.js', 'plugin:vue/recommended']
}




```

**What did you do? Please include the actual source code causing the issue.**

```vue
<style>
</style>

<template>
<div>
    <group>
        <cell title="title" value="value"></cell>
    </group>
</div>
</template>

<script>
import {Group, Cell} from 'vux'

export default {
    components: {
        Group,
        Cell
    }
}
</script>





```

**What did you expect to happen?**
no error.
**What actually happened? Please include the actual, raw output from ESLint.**
error  Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag

a simple repro can run `npm run lint` in [vux-demo](https://github.com/yonyou-cloud/vux-demo)

**Comment by Aladdin-ADD** [comment]
missing `/vue`. closing.

**Comment by stevgouws** [comment]
@Aladdin-ADD where were you missing that? I'm having the same issue and can't figure it out. Works with eslint-plugin-vue 2.1.0 but with 3.13.1 I get this error.

```
module.exports = {
  root: true,
  parserOptions: {
    sourceType: "module"
  },
  env: {
    browser: true,
    node: true
  },
  extends: [
    "prettier", 
    'plugin:vue/recommended'
  ],
  plugins: [
    "prettier",
  ],
  rules: {
    "no-debugger": process.env.NODE_ENV === "production" ? 2 : 0,
    "prettier/prettier": [
      "error",
      {
        semi: false,
        printWidth: 120
      }
    ]
  }
};
```

**Comment by Aladdin-ADD** [comment]
I'm not using https://github.com/yonyou-cloud/eslint-config, and I just missing `/vue` in config:
```js
// .eslintrc.js for vue.
module.exports = {
  extends: ['@yonyou-cloud/eslint-config/vue']
}
```

didn't see any problems in your config. maybe you can see more info using `eslint --debug`.

**Comment by stevgouws** [comment]
Cool thanks will check it out, otherwise will open new issue.

**Comment by phamhongphuc** [comment]
@stevgouws 
There is no need to install `@yonyou-cloud/eslint-config`

**Package.json**

    "eslint-plugin-vue": "^3.14.0",

**.eslintrc.json**

    "extends": [
        "plugin:vue/recommended"
    ],

**Comment by jeremyjh** [comment]
@stevgouws what was your solution to this problem?

**Comment by stevgouws** [comment]
@jeremyjh Don't think I solved it unfortunately. I think I went back to eslint-plugin-vue 2.1.0 for the time being, then a week later I switched back to Atom for other reasons.

**Comment by jeremyjh** [comment]
I just fixed all my issues with this: https://github.com/vuejs/eslint-plugin-vue/issues/186#issuecomment-329426553

**Comment by CamCode** [comment]
This could also fix your issue also the #186 
https://vuejs.github.io/eslint-plugin-vue/user-guide/#faq

**Comment by KrinalPatel11** [comment]
You should put your component between an enclosing tag, Which means:

// WRONG!

return (
<Comp1 />
<Comp2 />
)
Instead:

// Correct

return (
<div>
<Comp1 />
<Comp2 />
</div>
)
Edit: Per Joe Clay’s comment about the Fragments API

// More Correct

return (
<React.Fragment>
<Comp1 />
<Comp2 />
</React.Fragment>
)

[Here](https://icetutor.com/question/parse-error-adjacent-jsx-elements-must-be-wrapped-in-an-enclosing-tag/) Is Other Solution

**Comment by noobalex** [comment]
> This could also fix your issue also the #186
> https://vuejs.github.io/eslint-plugin-vue/user-guide/#faq

thx, that resolve my issue

**Comment by SimonSimCity** [comment]
If anyone else is wondering about the broken link - I think this where it should link to now: https://github.com/vuejs/eslint-plugin-vue/tree/master/docs/user-guide#question-faq
